docker/data-science/notebooks/ch06/3.spell-correction.ipynb

```diff
@@ -520,18 +520,18 @@
        "      <td>1</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th>137</th>\n",
-       "      <td>touxhpad</td>\n",
+       "      <th>136</th>\n",
```

Comment:
Seems we lost the sorting of the results?

Reply:
In all the history of 6.18, there's never been an output that lined up with the manuscript (checked 9 versions of the file). All the versions in the history were just random misspellings, none nearly as informative as tosheba/singin

I ultimately deemed the misspellings and corrections were intermediate data structures and not very informative independently. Even sorting the misspellings in all basic combinations I couldn't yield an informative set of candidates.

Without cherry picking or advanced sorting, I believe the printing of misspellings could just be omitted with a slightly tweaked post listing explanation.

Thoughts?

Reply:
I think this is 6.20, isn't it? It's already sorted by misspell count; issue is just there's no tie breaker. I think if it just has a secondary sort on the "misspell" column then the data will always be deterministic.

Reply:
Ah, I grouped this point of conversation with an outstanding issue regarding the mispelling data.

6.20 has a secondary sort now